src/server/rpc/procedures/geolocation/geolocation.js

```diff
@@ -58,7 +58,11 @@ if(!process.env.GOOGLE_GEOCODING_API) {
     };
 
 
-    // geocode an address and send back the details
+    /**
+     * geolocate and address and send back the details
```

Comment:
This description needs some work since these will eventually be sent back to the users: 
- capitalize the first letter
- I think "and" is a typo
-  what are the details? 

I would try something else like "Get a ___ from an address"